packages/ag-charts-community/src/module/optionsGraph.ts

```diff
@@ -246,7 +246,7 @@ export class OptionsGraph extends AdjacencyListGraph<unknown, string> implements
      * Resolve partial options against the existing graph at a given path without overriding the existing user values.
      * Returns an object with only those keys that were also present within `partialOptions`.
      */
-    resolvePartial(path: Array<string>, partialOptions?: PlainObject) {
+    resolvePartial(path: Array<string>, partialOptions?: PlainObject, opts?: { pick?: boolean }) {
```

Comment:
`pick` option makes sense. For context, this solves the issue with `strokeWidth` being dependent on `stroke` and needing to get back all the changed values in some uses. Compared to the other cases which wanted to not override the existing values.